ci-operator/step-registry/cucushift/installer/rehearse/gcp/ipi/disconnected/private/ovn/provision/cucushift-installer-rehearse-gcp-ipi-disconnected-private-ovn-provision-chain.yaml

```diff
@@ -11,7 +11,8 @@ chain:
   - ref: ipi-conf-telemetry
   - ref: ipi-conf-gcp
   - ref: ipi-conf-mirror
-  - ref: ovn-conf 
+  - ref: ovn-conf
+  - ref: ipi-conf-manual-creds-remove-unnecessary-creds
```

Comment:
I think this workflow does not need this step.

Reply:
Updated.